### DISCUSSION TITLE
Is there a hard limit to number of static pages vercel edge network (CDN) can allow?

### DISCUSSION THREAD

**Discussion by riceballsoup** [vercel/next.js#45590] [Help] [answered]
I understand there is a cache response size limit of 10mb (see image below) but aside from this I can't find anywhere that suggest Vercel CDN has a defined limit for total number of pages (or total size) "cache-able" on the Edge Network.

Can anyone please advise if Vecel Edge Network actually allows caching millions of static pages (without any limitation)? Thanks!

![image](https://user-images.githubusercontent.com/67973499/216837541-f046b95b-36d6-49f8-82df-ee261fdfaf25.png)

**Comment by ciruz** [comment, marked as answer]
I would contact Vercel support, the next.js discussion is the wrong place, for a question like this. There is no limit in the docs. But who knows.
